core/src/main/java/com/sequenceiq/cloudbreak/service/LoadBalancerConfigService.java

```diff
@@ -233,7 +236,7 @@ public boolean isLoadBalancerCreationConfigured(Stack stack, DetailedEnvironment
      * This method updates the JSON parameters associated with certain instance groups.
      *
      * @param availabilitySetPrefix A string prefix. Should be a stack name normally.
-     * @param loadBalancers the list of load balancers to look up target groups from.
+     * @param loadBalancers         the list of load balancers to look up target groups from.
```

Comment:
please remove tab

Reply:
This is in line with they style guide. it wasn't set previously

setting id="org.eclipse.jdt.core.formatter.alignment_for_parameters_in_constructor_declaration" value="16"